ledger/sandbox/src/main/scala/com/digitalasset/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -65,20 +65,9 @@ class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](initialState: => A
       submitter: Option[Party],
       transaction: Tx.CommittedTransaction,
       disclosure: Relation[Tx.NodeId, Party],
-      globalDivulgence: Relation[ContractId, Party],
+      divulgence: Relation[ContractId, Party],
       divulgedContracts: List[(Value.ContractId, ContractInst)])
     : Either[Set[RejectionReason], ALS] = {
-    // NOTE(RA): `globalImplicitDisclosure` was meant to refer to contracts created in previous transactions.
-    // However, because we have translated relative to absolute IDs at this point, `globalImplicitDisclosure`
-    // will also point to contracts created in the same transaction.
-    //
-    // This is dealt with as follows:
-    // - First, all transaction nodes are traversed without updating divulgence info.
-    //   - When validating a fetch/exercise node, both the set of previously divulged contracts and
-    //     the newly divulged contracts is used.
-    //   - While traversing consuming exercise nodes, the set of all contracts archived in this transaction is collected.
-    // - Finally, divulgence information is updated using `globalImplicitDisclosure` minus the set of contracts
-    //   archived in this transaction.
```

Comment:
This comment seems out of date.

Reply:
It's not, but it's not particularly important either. The `ActiveLedgerStateManager` tracks the state of the ledger over the course of a transaction but it's used only for the in-memory implementation and will soon be on its way out.